test/test_unit.py

```diff
@@ -573,13 +584,16 @@ def testStrictPadding(self):
             ("seq/foo_##.exr", []),
             ("seq/foo_@.exr", []),
             ("seq/foo_@@_extra.exr", []),
+            (u"seq/Фото_{left,right}.#.exr", [u"seq/Фото_left.1-3#.exr", u"seq/Фото_right.1-3#.exr"]),
+            (u"seq/Фото_{left,right}.@@@@.exr", [u"seq/Фото_left.1-3#.exr", u"seq/Фото_right.1-3#.exr"]),
+            (u"seq/Фото_{left,right}.@@@.exr", []),
```

Comment:
For the unicode file tests, should we have it generate the test files as part of the test setup/teardown so that we can avoid packaging problems?  Or a second option would be to do the unicode tests separately using `yield_sequences_in_list()` so that files aren't needed.

Reply:
I'm not familiar with Appveyor - is it ok with tests spitting out random files?

Reply:
It just boots a temporary vm/container so I imagine you just write to tempdir and it should be fine.